releasenotes/notes/1.3/deprecate-basic-simulator-configuration-9d782925196993e9.yaml

```diff
@@ -25,8 +25,8 @@ deprecations_providers:
       backend.configuration().memory             No representation      
       backend.configuration().max_shots          No representation      
 
-    (*) Note that ``backend.target.operation_names`` includes ``basis_gates`` and additional 
+    (*) Note that :meth:`.Backend.target.operation_names` includes ``basis_gates`` and additional 
```

Comment:
This isn't correct and won't resolve correctly. It would be ``:attr:` instead of meth because target is an attribute of backend. But sphinx won't be able to resolve this because target isn't a path it can find as it's an instance attribute. This was probably more correct before.